src/app/tab/covid/covid.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="covidContainer" [style.padding]="getWidth(settings.config.covidData.countries[index].padding)">
+  <div class="flag" *ngIf="settings.config.covidData.countries[index].enableFlag"
+  >
+    <img  [ngStyle]="{ 'height' :shared.getImageSize(settings.config.covidData.countries[index].flagSize)}"  [src]="countryData.countryInfo.flag" alt="flag" />
+  </div>
+  <div class="info" [style.padding]="getWidth(settings.config.covidData.countries[index].padding)">
```

Comment:
You don't need to set padding here, setting the "gap" property above as mentioned will work fine.